styles.css

```diff
@@ -0,0 +1,117 @@
+body {
+  background-image: url('header-shapes mobile@2x.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+header { /* HEADER CONTAINER */
+  display: flex;
+  height: 7vh;
+  justify-content: space-between;
+  margin-right: 6.4%;
+}
+
+.nav {
+  display: flex;
+  margin: 20%;
+}
+
+img {
+  max-width: 100%;
+}
+
+.menu {
+  /* border: 1px solid red; */
+  align-content: center;
+  justify-content: center;
+}
+
+.logo {
+  color: #6070ff;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  width: 26.666666%;
+}
+
+a {
+  /* border: 1px solid red; */
+}
+
+.nav-bar {
+  display: flex;
+  width: 10.666666%;
+  justify-content: center;
+}
+
+.logo,
+.nav-bar {
+  /* border: 1px solid red; */
+  justify-content: center;
+}
+
+.app-wrap {
+  display: flex;
+  flex-direction: column;
+}
+
+.salute {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 130%;
+  line-height: 52px;
+}
+
+.description {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 150%;
+  color: #344563;
+}
+
+.lets_connect {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
+  color: #7f8cff;
+}
+
+.social-links {
+  display: flex;
+  width: 45%;
+  direction: flex;
+}
+
+.social {
+  margin: 1%;
+  align-content: space-around;
+}
+
+.text_logo {
+  margin: 4px;
+}
+
+.content { /* CONTENT CONTAINER */
+  /* border: 1px solid red; */

```

Comment:
- [ ] Please 🙏🏽 remove the commented line when it represents an unnecessary piece of code for the best practices of HTML & CSS.